discovery-infra/test_infra/consts.py

```diff
@@ -27,6 +27,11 @@
 DEFAULT_ADDITIONAL_NTP_SOURCE = "clock.redhat.com"
 
 
+all_host_stages = ['START_INSTALLATION', 'INSTALLING', 'WRITE_IMAGE_TO_DISK',
```

Comment:
You should declare it after the `HostsProgressStages` class and use its values.
i.e.
```python
HOSTS_STAGES = [HostsProgressStages.START_INSTALLATION, HostsProgressStages.INSTALLING, ...]
```

We can even do a smarter usage of the `enum` class and convert the class into an enum, then iterating upon it.

```python
    from enum import Enum
    class HostsProgressStages(Enum):
        ...

    stages_values = list(map(lambda stage: stage.value ,HostsProgressStages))
    index = stages_values.index(stage)
    ....
```

But it might break other functions use this class.